hushline/routes.py

```diff
@@ -120,15 +122,37 @@ def inbox() -> Response | str:
             is_personal_server=app.config["IS_PERSONAL_SERVER"],
         )
 
+    def verify_hcaptcha(hcaptcha_response: str) -> bool:
+        hcaptcha_secret = os.getenv("HCAPTCHA_SECRET_KEY")
+        verify_url = "https://hcaptcha.com/siteverify"
+        data = {"secret": hcaptcha_secret, "response": hcaptcha_response}
+
+        try:
+            response = requests.post(verify_url, data=data, timeout=5)
+            result = response.json()
+            return result.get("success", False)
+        except requests.RequestException as e:
+            app.logger.error("hCaptcha verification error: %s", str(e), exc_info=True)
+            return False
+
     @app.route("/submit_message/<username>", methods=["GET", "POST"])
-    def submit_message(username: str) -> Response | str:
+    def submit_message(username: str) -> Union[Response, str]:
         form = MessageForm()
         user = User.query.filter_by(primary_username=username).first()
         if not user:
-            flash("🫥 User not found.")
+            flash("User not found.")
             return redirect(url_for("index"))
 
+        is_personal_server = os.getenv("IS_PERSONAL_SERVER", "false").lower() == "true"
```

Comment:
Same here. Instead of directly loading the environment, you can just do:

```py
is_personal_server = app.config["IS_PERSONAL_SERVER"]
```

Reply:
This is also nice because you don't need the like `"false").lower() == "true"` stuff each time